cypress/support/commands.ts

```diff
@@ -148,7 +148,7 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add('enableShowMultipleInDebugTree' as keyof Chainable, () => {
-  cy.get('[data-cy-open-settings-modal]').click();
-  cy.get('input[data-cy-toggle-show-amount]').click();
-  cy.get('[data-cy-settings-modal-save-changes]').click();
+  cy.get("[data-cy-debug-table='openSettings']").click();
+  cy.get("[data-cy-settings-modal='showAmount']").click();
+  cy.get("[data-cy-settings-modal='saveChanges']").click();
```

Comment:
Modal mag je hier weglaten
```suggestion
  cy.get("[data-cy-settings='showAmount']").click();
  cy.get("[data-cy-settings='saveChanges']").click();
```